### PR TITLE
[ios] 12791 add unit tests public

### PR DESCRIPTION
**Pull request by adam-govan**
@dominicmarmionswrve  / @seaders 

some very small tests but it's the project I'd like to get started with.

### CODE REVIEW

File: SwrveDemo/Podfile.lock [hunk 1]
```diff
@@ -0,0 +1,29 @@
+PODS:
```
Comment (dominicmarmionswrve) [inline SwrveDemo/Podfile.lock:1]: Lets not add Podfile.lock files to github

Comment (adam-govan) [inline SwrveDemo/Podfile.lock:1]: whoops! good call

**Comment by dominicmarmionswrve** [comment]
🚀  🎸  👯  🎈

File: SwrveDemo/Podfile.lock (hunk 1 above)
Comment (dominicmarmionswrve) [reply SwrveDemo/Podfile.lock:1]: @adam-govan Is this Podfile.lock supposed to be here?

**Pull request merged**